### ISSUE TITLE
[TW-1189] bash_completion needs updating

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1224]
_John Florian on 2009-06-16T16:47:12Z says:_ 

 There are still references to taskprogram(at)googlegroups.com in bash_completion.sh as of 1.7.1.

Also steps 1 and 2 as described in the same file should probably use underscores instead of hyphens to match the packaged file.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2009-06-16T16:47:12Z
Modified: 2014-02-09T02:39:08Z
```

**Comment by taskwarrior** [comment]
_Federico Hernandez on 2009-06-16T17:48:37Z says:_ 

 This has already been fixed in the upcoming 1.8.0. As well as automatic installation on Fedora.
